UnitTests/UnitTests.UWP/UI/Controls/Test_RichSuggestBox.cs

```diff
@@ -0,0 +1,272 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Windows.UI.Text;
+using Microsoft.Toolkit.Uwp;
+using Microsoft.Toolkit.Uwp.UI.Controls;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTests.UI.Controls
+{
+    [TestClass]
+    public class Test_RichSuggestBox : VisualUITestBase
+    {
+        [TestCategory(nameof(RichSuggestBox))]
+        [TestMethod]
+        [DataRow("@Token1", "@Token2", "@Token3")]
```

Comment:
Can we maybe try temporarily removing the datarows here and below and see if that makes any difference?